recipe/meta.yaml

```diff
@@ -24,10 +24,9 @@ requirements:
     - pkg-config
     - ninja
   host:
-    - boost-cpp
+    - libboost-devel
```

Comment:
Given that `websocketpp` is header-only, we should probably just depend on `libboost-headers` here?

Reply:
Good catch, I was not aware of this. I will check this tomorrow (european) morning, thanks!

Reply:
Do you have a pointer to some description of what the different boost new packages contain? Thanks!

No problem, I found it in https://github.com/conda-forge/boost-feedstock/pull/164 .

Reply:
Technically the modification is correct, but that would mean that by just installing `websocketpp` users could not call `find_package(websocketpp REQUIRED)` in CMake out of the box. Could it make sense to add a `libwebsocketpp-devel` that also installs `libboost-devel` ?

Reply:
If websocketpp is only usable with the boost metadata, then it's fine to keep it. I just thought I'd flag it.